src/auto.ts

```diff
@@ -111,6 +111,13 @@ export type AutoloadConfig = {
      * @experimental
      */
     enableExperimentalMetadata?: boolean;
+
+    /**
+     * Enable ABI helpers to apply additional transformations to the ABI before returning.
+     * @group Settings
+     * @experimental
+     */
+    abiHelpers?: Array<(abi: ABI) => ABI>;
```

Comment:
Sure, let's remove it from autoload: 32be43b2d3b525177bba37196871232701fd4a39

Also renamed the function: 2ba3e9e366c27e62c6b8cb7611faa3ebbf76d990

We are using `enableExperimentalMetadata` but not really sure if we need it at the moment.

Reply:
I'd feel more comfortable making it default under the `enableExperimentalMetadata` flag, since I don't think many people use it so it won't be very breaking (plus it is experimental).

Also I think WhatsABI would only generate types with empty names with that flag enabled? Or I might be mistaken.

Reply:
Appreciate the changes. No need to make further changes, just brainstorming at the moment. :)